src/lib/Form/Data/Location/LocationCopySubtreeData.php

```diff
@@ -15,7 +16,9 @@ class LocationCopySubtreeData extends AbstractLocationCopyData
      * @var \Ibexa\Contracts\Core\Repository\Values\Content\Location|null
      *
      * @AdminUiAssert\LocationIsWithinCopySubtreeLimit()
+     *
      * @AdminUiAssert\LocationIsNotRoot()
+     *
```

Comment:
I am not sure if we want to separate it like this. In the product catalog, we have another approach

Reply:
Fixed in c1a8e9f4.

Reply:
Nevermind, it seems the issue is more complex than it initially looked like.

Newer versions of php-cs-fixer have `phpdoc_separation` available as an array of tags that should be grouped together. Because we're setting it to `true` in our config, we overwrite the defaults and this is the result.